Chapters/3-9-Interlogue.md

```diff
@@ -0,0 +1,51 @@
+
+"Two days and Four va'la... since launch."
+
+"This is my first" _last_ "... first log since radio silence. I passed through the obscura with only no deviation in my trajectory. It was like breaking through a delicate piece of fabric: I barely knew it was there. Some of it stuck to my craft; I can see it on the windows, but it doesn't seem to be doing any harm."
+
+"Of course, you know by now, that the obscura is surrounding all of Ara like a shell. Even if we could blast away the portion nearest us, it would rotate and rebuild and we would be right back to where we were. We never stood a chance."
+
+Thi adjusted haeself in the craft.
+
+"When I look out the window, All I see is light. I can't look for very long. It's so bright."
+
+"I had to push the walls to orient the window away from Ara. But the obscura is still... it's everywhere I can see and incredibly bright. A pale red-infrared sky below me."
+
+"It's beautiful in a way... In it's own way. I never knew something so destructive could evoke such beauty. I'm not sure these things understand what beauty is. I don't imagine they care."
+
+Thi paused and drew a gulp of water from the water tank. It cooled her head and throat.
+
+"The obscura... I don't know of a good way to describe them other than thin rocks that seem to be assembling themselves into extremely flat and thin surfaces. I can't imagine how it might do such a thing. As to whether this is natural or not, I believe that is so far beyond our understanding that the question is meaningless. In my prior logs, I talked about communication, what I might say to these imaginary beings that we made up on the ground, how I might learn their language, form a connection. It seems so silly now, but I think there may be something. Not through sounds, or movements. But through something else. Maybe they can hear me now, speaking into the void."
+
+A sigh and a sniffle rattled the air as thi wiped a tear from hae face.
+
+"Whoever is listening to this... I want you to know that I'm not afraid of what comes next."
+
+"My nather taught me to befriend my fear, so I know when it's betraying me. But I don't feel it now. I know that I should be afraid. It's strange that I'm not. But this place, this vast sea of light: it makes me feel at ease. I know that I will be remembered. That's all anyone can ask for when they leave."
+
+A high-pitched twinkling sound entered hae sound field. The binding pendant around hae neck was floating in front of hae.
+
+"We were just regular people... We got swept up in the chaos of it all. Kipnu and Melna and me. I could say that we didn't deserve to die. But doesn't everyone deserve to live? We stuck our heads where others buried theirs. I'll never know if our efforts mattered. But honestly, I don't give a damn. We tried."
+
+"...Well," thi dotted the silence, but nothing else came to hae.
+
+"It's getting rather hot up here. Think I'll spare you the details."
+
```

Comment:
Oh wow. I think even if you end your book at part 3. You should keep this as an epilogue, it feels like a great follow and resolution to what just happened last chapter.